twitter4j-core/src/main/java/twitter4j/User.java

```diff
@@ -46,6 +46,13 @@
     String getScreenName();
 
     /**
+     * Returns true if it's following the logged in user
```

Comment:
Suggested improved wording:- "Returns true if the authenticated user is following this user"